components/raftstore/src/store/fsm/peer.rs

```diff
@@ -2646,6 +2646,7 @@ where
         );
 
         let msg_type = msg.get_message().get_msg_type();
+        #[cfg(feature = "failpoints")]
```

Comment:
why?

Reply:
To handle build warning as fp_enable is only used in failpoint.